error if step_dummy produce too big data.frame (#1327)
tidymodels/recipes: NEWS.md

* The `prefix` argument of `step_dummy_multi_choice()` is not properly documented. (#1298)

> * `step_dummy()` now give an informative error if it tried to generate too many columns to fit in memory. (#828)

Reviewer: ```suggestion
* `step_dummy()` now gives an informative error on attempt to generate too many columns to fit in memory. (#828)
```
